coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -12,6 +12,7 @@ class DocumentationComment:
     inside source-code, like position etc.
     """
     Parameter = namedtuple('Parameter', 'name, desc')
+    ExceptionValue = namedtuple('ExceptionValue', 'name, desc')
```

Comment:
Well, technically you could. But it would create nothing but confusion. :stuck_out_tongue: